### DISCUSSION TITLE
Please add save to bookmark button on card itself rather than a menu option

### DISCUSSION THREAD

**Discussion by sonawaneapekshit** [dailydotdev/daily#654] [Feature Requests]
In the current version of daily.dev the bookmark option is in the menu and I think the save to bookmark should be a button instead of a menu option.

Thank you Daily dev team for creating a great app for devs

**Comment by idoshamun** [comment]
Hi @sonawaneapekshit,

This is a dup of https://github.com/dailydotdev/daily/discussions/638. Let's keep the discussion there :)

**Comment by nimrodkra** [comment]
@sonawaneapekshit the bookmark button is back on the card - thanks for the feedback 🙏

![IMG_8444](https://github.com/dailydotdev/daily/assets/41470823/f721126d-c097-43a6-90ff-267864e6eef9)
